packages/react-storage/src/components/StorageBrowser/composables/FolderNameField.tsx

```diff
@@ -12,7 +12,7 @@ export interface FolderNameFieldProps {
   placeholder?: string;
   onChange?: (value: string) => void;
   onValidate?: (value: string) => void;
-  validationMessage?: React.ReactNode;
+  validationMessage?: string;
```

Comment:
It appeared we were only setting this as a string in our code (in tests we used a React Node). The Amplify UI Field components expect a string for errorMessage (although it could be loosened to take a React node and then this change wouldn't be necessary). Would love thoughts

Reply:
Intentionally have it typed as `React.ReactNode` for flexibility, Preference is to loosen the type on the Amplify UI Field